cmd/metal-api/internal/service/v1/tenant.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+import "google.golang.org/protobuf/types/known/timestamppb"
+
+type (
+	TenantGetHistoryRequest struct {
+		At *timestamppb.Timestamp
```

Comment:
Timestamppb is for protobuf. For this API you should just use time.Time.

Explicit json annotation + description would be great as this will be rendered into the swagger spec. Also you need to check in the generated swagger spec.